packages/react-core/src/components/SimpleList/SimpleListItem.tsx

```diff
@@ -4,6 +4,8 @@ import styles from '@patternfly/react-styles/css/components/SimpleList/simple-li
 import { SimpleListContext } from './SimpleList';
 
 export interface SimpleListItemProps {
+  /** id for the item. */
+  itemId?: number | string;
```

Comment:
Can we just call this prop `id`?

Reply:
not sure, I would rather not to avoid confusion with HTML `id` property. It's the same as in the `NavItem` component.